_episodes/01-intro.md

```diff
@@ -220,4 +220,28 @@ weight in kilograms is now: 65.0
 > {: .solution}
 {: .challenge}
 
+>
```

Comment:
We actually wanted to keep your exercise title `> ## Seeing Data Types`. Please bring it back :).